### PR TITLE
fix(node): use non-deprecated stream names for REPLServer

### PR DESCRIPTION
**Pull request by addaleax**
See https://nodejs.org/api/deprecations.html#DEP0141

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` should be present and it shouldn't have any additional or disabling of rules. Just content as `{ "extends": "dtslint/dt.json" }`. If for reason the some rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]`  and not for whole package so that the need for disabling can be reviewed.
- [ ] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://nodejs.org/api/deprecations.html#DEP0141
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/node/repl.d.ts [hunk 1]
```diff
@@ -136,13 +136,21 @@ declare module "repl" {
          */
         readonly context: Context;
         /**
-         * The `Readable` stream from which REPL input will be read.
+         * @deprecated since v14.3.0 - Use `input` instead.
          */
         readonly inputStream: NodeJS.ReadableStream;
         /**
-         * The `Writable` stream to which REPL output will be written.
+         * @deprecated since v14.3.0 - Use `outputt` instead.
```
Comment (ExE-Boss) [inline types/node/repl.d.ts:143]:
```suggestion
         * @deprecated since v14.3.0 - Use `output` instead.
```

Comment (addaleax) [reply types/node/repl.d.ts:143]: @exe-boss thanks for catching, done :)

**Review by ExE-Boss — APPROVED** [review]

**Review by a-tarasyuk — APPROVED** [review]

**Review by Flarna — APPROVED** [review]

**Comment by addaleax** [comment]
I have no idea why CI failed here, to be clear. None of the failures seem to be related.

**Comment by Flarna** [comment]
Try rebase/merge from master to retrigger CI.

**Comment by addaleax** [comment]
(For context, the rebase was just to get rid of flaky CI failures, no actual changes were made)

**Review by Flarna — APPROVED** [review]

**Review by RyanCavanaugh — APPROVED** [review]

**Comment by addaleax** [comment]
Ready to merge

**Pull request merged**